client/keystore/Cargo.toml

```diff
@@ -14,6 +14,11 @@ rand = "0.7.2"
 serde_json = "1.0.41"
 subtle = "2.1.1"
 parking_lot = "0.10.0"
+sp-keyring = { version = "2.0.0", path = "../../primitives/keyring", optional = true }
+tempfile = "3.1.0"
 
 [dev-dependencies]
-tempfile = "3.1.0"
+
+[features]
+default = []
+test-helpers = ["sp-keyring"]
```

Comment:
Please revert all these changes.

No `test-helpers` features! This leaks into main build.